compiler/src/dotty/tools/dotc/decompiler/PartialTASTYDecompiler.scala

```diff
@@ -0,0 +1,12 @@
+package dotty.tools.dotc.decompiler
+
+import dotty.tools.dotc.core.Phases.Phase
+
+/** Partial TASTYDecompiler that doesn't execute the backendPhases
+  * allowing to control decompiler output by manually running it
+  * on the CompilationUnits
+ */
+class PartialTASTYDecompiler extends TASTYDecompiler {
+  override def phases: List[List[Phase]] =
```

Comment:
Do `override protected def backendPhases: List[List[Phase]] = Nil` instead of this.